### PR TITLE
minor readme spelling and grammar fixes

### PR DESCRIPTION
**Pull request by transitive-bullshit**
This module is super helpful @TrySound -- thanks for releasing it!

I just wanted to make some small English edits & fixes to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,14 +45,14 @@ export default {
...
+type: `string`
 default: `'.size-snapshot.json'`
 
 ### matchSnapshot
 
-This option allows to check that contributor do not forget to build or to commit `.size-snapshot.json` file. If this is `true` the plugin will match existing snapshot and the new one.
+This option allows you to verify that contributors don't forget to build or commit the `.size-snapshot.json` file. If this is `true`, the plugin will validate the snapshot against an existing one.
 
-type: `boolean`  
+type: `boolean`
```
Comment (TrySound) [inline README.md]: Could you please keep these spaces? They are used in markdown to insert `<br />`.

File: README.md [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 
-This provides the information about
+This plugin provides details about
 
-* actual bundle size which will consume user bundler
-* minified bundle size which will parse browsers in production
-* gzipped size which will be loaded in production
+* actual bundle size
+* minified bundle size
+* gzipped bundle size
```
Comment (TrySound) [inline README.md]: I would like somehow to describe what these three sizes mean for end user and developer. [Here's reference](https://github.com/TrySound/rollup-plugin-size-snapshot/issues/15). Would be great if you will help with this.

File: README.md (hunk 1 above)
Comment (transitive-bullshit) [reply README.md]: sounds good -- my editor is set to automatically remove trailing whitespaces, so this was on accident :)

**Comment by transitive-bullshit** [comment]
Your english is a lot better than my russian haha :)

**Comment by transitive-bullshit** [comment]
Happy to make any more english changes here or to other packages -- just let me know 👍

File: README.md [hunk 3]
```diff
@@ -5,9 +5,9 @@
 
 This plugin provides details about
 
-* actual bundle size
-* minified bundle size
-* gzipped bundle size
+* actual bundle size (approx runtime size)
```
Comment (TrySound) [inline README.md]: Not sure what `runtime` meant in that tweet. I'd call it "bundler parsing size" similar to the next point.

**Comment by transitive-bullshit** [comment]
Ahhhhh, when I made that last fix, it reverted my space changes because of my editor... hold on a sec

**Comment by TrySound** [comment]
Thank you, Travis.

**Pull request merged**